### PR TITLE
Adds missing SCIM link

### PR DESCRIPTION
**Pull request by jmelberg-okta**
## Description:
- Fixes [missing CRUD link](https://github.com/okta/okta.github.io/compare/jm-add-missing-crud-link?expand=1#diff-4d4d3bd54f55bfc7712bcd9d8ef4b60eR1052) and fixes markdown lint issues.

### Resolves:
* #1879

### CODE REVIEW

File: _source/_standards/SCIM/index.md [hunk 1]
```diff
@@ -125,14 +126,11 @@ Until then, consider out-of-band processes that work around these use cases.
 
 * Attribute-level mastering&mdash;The app wants to be the master for some employee attributes like phone number, while letting Okta or another app master other attributes. We call this attribute-level mastering.
```
Comment (jakubvul) [inline _source/_standards/SCIM/index.md]: Can we use this as an opportunity to get rid of these em dashes?

File: _source/_standards/SCIM/index.md [hunk 2]
```diff
@@ -739,17 +731,13 @@ With Group Push Beta, Okta now supports creation of a Group along with its user
 
 With Group Push Beta, Okta now supports reading the Group's details by group id along with the membership details. If a Group is not found, your SCIM application may return a HTTP status 404("not found").For more details on the `/Groups/{id}` SCIM endpoint, see [section 3.4.1](https://tools.ietf.org/html/rfc7644#section-3.4.1) of the [SCIM 2.0 Protocol Specification](https://tools.ietf.org/html/rfc7644).
 
-
 ##### Update Group Details: PUT /Groups/{id}
```
Comment (jakubvul) [inline _source/_standards/SCIM/index.md]: Again, minor, but this should be "/groups" throughout, not "/Groups".

File: _source/_standards/SCIM/index.md [hunk 3]
```diff
@@ -1283,10 +1224,10 @@ Yes, you can republish your app, but we recommend you don’t do it frequently.
...
 ### Dev Doc Examples
 
-Box - https://docs.box.com/docs/getting-started-box-integration
-Slack - https://api.slack.com/
-OneLogin - https://developers.onelogin.com/
-Zendesk - https://developer.zendesk.com/apps
+Box - [https://docs.box.com/docs/getting-started-box-integration](https://docs.box.com/docs/getting-started-box-integration)
+Slack - [https://api.slack.com/](https://api.slack.com/)
+OneLogin - [https://developers.onelogin.com/](https://developers.onelogin.com/)
+Zendesk - [https://developer.zendesk.com/apps](https://developer.zendesk.com/apps)
```
Comment (jakubvul) [inline _source/_standards/SCIM/index.md]: If I'm not mistake, you can just put `<>` around the URLs and that will accomplish the same. 

e.g. `<https://docs.box.com/docs/getting-started-box-integration>`

Comment (jmelberg-okta) [reply _source/_standards/SCIM/index.md]: Oh man. I completely forgot about this. Thanks!

**Pull request merged**